src/pages/hire.js

```diff
@@ -100,11 +102,11 @@ class HirePage extends Component {
             the cooperative community at Founders and Coders. We are always
             looking for new partners committed to building inclusive workplaces
             where our diverse graduates will thrive. Think your company would be
-            a good fit? <_ExternalLink href="https://docs.google.com/forms/d/e/1FAIpQLScqtbiYqT7CDDjjYFtcwzlWDuZlsotcDvP1SoSj2SmuVmO4AA/viewform">Fill out this short form</_ExternalLink>.
+            a good fit? <_ExternalLink href={HIRE_CONTACT_FORM}>Fill out this short form</_ExternalLink>.
           </HeadingWithBody>
           <DoubleLine colour="red" />
-          <section className="mb7 pb5 mh2 mh0-ns">
-            <FlickityCarousel
+            <SplashPhoto src={hireSplash} />
+            {/* <FlickityCarousel
```

Comment:
Can you remove this?

Reply:
Sorry what should I remove? The commented out code?